### PR TITLE
WordPress security settings

### PR DESCRIPTION
**Pull request by luboskmetko**
Various WP settings based on #172

### CODE REVIEW

File: generators/wp/templates/wp-config.php [hunk 1]
```diff
@@ -101,9 +102,17 @@
...
 if ( !defined('ABSPATH') )
-	define('ABSPATH', dirname(__FILE__) . '/');
+	define( 'ABSPATH', dirname(__FILE__) . '/' );
 
 /** Sets up WordPress vars and included files. */
-require_once(ABSPATH . 'wp-settings.php');
+require_once( ABSPATH . 'wp-settings.php' );
+
+/** Disable the Plugin and Theme Editor */
+define( 'DISALLOW_FILE_EDIT', true );
```
Comment (JakubSzajna) [inline generators/wp/templates/wp-config.php]: Why don't we add this in else clause so this can be easily changed in wp-config-local.php?

**Comment by JakubSzajna** [comment]
@luboskmetko I've added one comment in the config.php file. Maybe we should change place of this config so it can be overridden in wp-config-local if needed?

Everything else looks good.

**Comment by luboskmetko** [comment]
@JakubSzajna can you check the last commit if that's what you meant? Thanks

**Comment by JakubSzajna** [comment]
Yes, this is what I asked for, thanks!

**Comment by jakub300** [comment]
Hey @luboskmetko,

I see that you added code between `# BEGIN/END WordPress` comments, could you verify that going `Settings → Permalinks` and clicking `Save` will not overwrite that?

**Comment by luboskmetko** [comment]
@jakub300 good point, it does. I'll fix it. Thanks!

**Pull request merged**